typescript/packages/lookslike-high-level/src/recipes/coder.ts

```diff
@@ -0,0 +1,87 @@
+import { html } from "@commontools/common-html";
+import { recipe, UI, NAME, handler, } from "@commontools/common-builder";
+import ts from 'typescript';
+import * as commonHtml from "@commontools/common-html";
+import * as commonBuilder from "@commontools/common-builder";
+import * as commonRunner from "@commontools/common-runner";
+import * as commonData from "../data.js";
+
+const runCode = handler<{}, { src: string, output: string }>(
+    (_, state) => {
+        // Custom module resolution
+        const customRequire = (moduleName: string) => {
+            switch (moduleName) {
+                case "@commontools/common-html":
+                    return commonHtml;
+                case "@commontools/common-builder":
+                    return commonBuilder;
+                case "@commontools/common-runner":
+                    return commonRunner;
+                case "../data.js":
```

Comment:
this makes me want to put launch somewhere else... but it is probably too early

Reply:
we should probably treat it like any number of "built-ins", which we should move out of being hard coded in builder and available as packages to import, like many other reusable lifted functions and recipes. Let's do that this week, maybe when replacing generateData?

Reply:
we could for now at least change this string to be "@comontools/common-<...>"?